pkg/types/intoto/v0.0.2/intoto_v0_0_2_schema.json

```diff
@@ -0,0 +1,103 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/intoto/intoto_v0_0_2_schema.json",
+    "title": "intoto v0.0.2 Schema",
+    "description": "Schema for intoto object",
+    "type": "object",
+    "properties": {
+        "content": {
+            "type": "object",
+            "properties": {
+                "envelope": {
+                    "description": "dsse envelope",
+                    "type": "object",
+                    "properties": {
+                        "payload": {
+                            "description": "payload of the envelope",
+                            "type": "string",
+                            "format": "byte",
+                            "writeOnly": true
+                        },
+                        "payloadType": {
+                            "description": "type describing the payload",
+                            "type": "string"
+                        },
+                        "signatures": {
+                            "description": "collection of all signatures of the envelope's payload",
+                            "type": "array",
+                            "minItems": 1,
+                            "items": {
+                                "description": "a signature of the envelope's payload along with the public key for the signature",
+                                "type": "object",
+                                "properties": {
+                                    "keyid": {
+                                        "description": "optional id of the key used to create the signature",
+                                        "type": "string"
+                                    },
+                                    "sig": {
+                                        "description": "signature of the payload",
+                                        "type": "string",
+                                        "format": "byte"
+                                    },
+                                    "publicKey": {
+                                        "description": "public key that corresponds to this signature",
+                                        "type": "string",
+                                        "format": "byte",
+                                        "readOnly": true
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "required": ["payloadType", "signatures"],
+                    "writeOnly": true
```

Comment:
this shouldn't affect the code generation, but I would remove this since it's incorrect
```suggestion
```

Reply:
changed.